appinfo/application.php

```diff
@@ -20,72 +20,40 @@
  * application settings are registered
  */
 class Application extends App {
-    const APP_ID = 'agreedisclaimer';
-
-    /** Preffix used for naming the txt and pdf files */
-    const FILE_PREFFIX = 'disclaimer';
-
-    /** Maximum file size in megabytes */
-    const FILE_SIZE_LIMIT = 3;
-
     /**
      * Creates an Application object and registers its related services,
-     * user hooks, and settings 
+     * user hooks, and settings
      */
     public function __construct(array $urlParams=array()) {
-        parent::__construct(self::APP_ID, $urlParams);
-        $this->registerServices();
-        $this->registerHooks();
-        $this->registerSettings();
-    }
-
-    /**
-     * Registers all the application services
-     */
-    public function registerServices() {
+        parent::__construct('agreedisclaimer', $urlParams);
         $container = $this->getContainer();
+        // register parameters
+        $container->registerParameter('fileSizeLimit', 3);
```

Comment:
Registering parameters will make them usable for injecting them by variable name